types/timestamp.go

```diff
@@ -31,3 +31,17 @@ func (ts TimestampSlice) Less(i, j int) bool {
 func (ts TimestampSlice) Swap(i, j int) {
 	ts[i], ts[j] = ts[j], ts[i]
 }
+
+// Clock allows clients to retrieve the current time.
+type Clock interface {
+	Now() Timestamp
+}
+
+// StdClock is an implementation of Clock that retrieves the current time using
+// the system time.
+type StdClock struct{}
+
+// Now retrieves the current timestamp.
+func (c StdClock) Now() Timestamp {
+	return Timestamp(time.Now().Unix())
+}
```

Comment:
> You make a clock, but you don't use it in any tests.

Can you clarify what you mean? `StdClock` does not have any unit tests because I don't think it's possible to unit test it (open to suggestions if you guys see a way though). It does have test coverage because `ConsensusSet` uses it and those tests hit `StdClock`.